packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -430,8 +430,8 @@ module.exports = async ({
    */
   plugins.extractText = options =>
     new MiniCssExtractPlugin({
-      filename: `../css/[name].[contenthash].css`,
-      chunkFilename: `../css/[name].[contenthash].css`,
+      filename: `[name].[contenthash].css`,
```

Comment:
I've added back the `../css/` back as this wasn't supposed to be in this commit. See the following commit for reference.